src/core/core.typedRegistry.js

```diff
@@ -43,7 +43,7 @@ export default class TypedRegistry {
 		}
 
 		if (Object.keys(defaults.get(scope)).length) {
-			throw new Error('Can not register "' + id + '", because "defaults.' + scope + '" would collide with existing defaults');
+			throw new Error(`Can not register "${id}", because "defaults.${scope}" would collide with existing defaults`);
```

Comment:
Internet explorer yes, but we are using babel for that. 
dist/chart.js:
```js
      throw new Error("Can not register \"" + id + "\", because \"defaults." + scope + "\" would collide with existing defaults");
```